tests/test_match_probabilistic_task.py

```diff
@@ -26,8 +26,8 @@
 import lsst.afw.geom as afwGeom
 from lsst.meas.astrom import ConvertCatalogCoordinatesConfig, MatchProbabilisticConfig, MatchProbabilisticTask
 
+import astropy.table
```

Comment:
Unfortunately, we need to test the deprecated pandas usage as well, but catching the deprecation warnings.